File: 0448-find-all-numbers-disappeared-in-an-array/0448-find-all-numbers-disappeared-in-an-array.cpp
class Solution {
public:
    vector<int> findDisappearedNumbers(vector<int>& nums) {
        
        unordered_set<int> st;
        vector<int> op;

        for(int i=0;i<nums.size();i++)
        {
            st.insert(nums[i]);
        }

        for(int i=1;i<=nums.size();i++)
        {
            if(!st.contains(i))
            {
                op.push_back(i);
            }
        }

        return op;
    }
};

File: 0448-find-all-numbers-disappeared-in-an-array/0448-find-all-numbers-disappeared-in-an-array.py
class Solution:
    def findDisappearedNumbers(self, nums: List[int]) -> List[int]:

        for i in range(len(nums)):
            ind = abs(nums[i]) - 1

            if nums[ind] > 0:
                nums[ind] = nums[ind]*(-1)

        op = []
        for i in range(len(nums)):
            if nums[i] > 0:
                op.append(i+1)

        return op

        
        
